bip-0324.mediawiki

```diff
@@ -0,0 +1,467 @@
+<pre>
+  BIP: 324
+  Layer: Peer Services
+  Title: Version 2 Peer-to-Peer Message Transport Protocol
+  Author: Jonas Schnelli <dev@jonasschnelli.ch>
+  Status: Draft
+  Type: Standards Track
+  Created: 2019-03-08
+  License: PD
+</pre>
+
+== Abstract ==
+
+This BIP describes a new Bitcoin peer to peer transport protocol with opportunistic encryption.
+
+== Motivation ==
+
+The current peer-to-peer protocol is partially inefficient and in plaintext.
```

Comment:
Maybe link to the section "Length comparisons between v1 and v2 messages" as a reference for the inefficiency.